.github/workflows/release-preparations.yml

```diff
@@ -0,0 +1,54 @@
+name: Release preparations
+
+on:
+  workflow_dispatch:
+    inputs:
+      release:
+        default: patch
+        description: SemVer release level
+        options:
+          - major
+          - minor
+          - patch
+        required: true
+        type: choice
+
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  aeruginous:
+    if: |
+      contains(fromJson('["major", "minor", "patch"]'), inputs.release) &&
+      github.repository == 'sourcefrog/cargo-mutants'
+    runs-on: ubuntu-latest
+    steps:
+      - run: rustup update
+      - uses: actions/checkout@v4
+      - uses: fregante/setup-git-user@v2
+
+      - uses: baptiste0928/cargo-install@v2
+        with:
+          crate: aeruginous
+
+      - run: |
+          aeruginous increment-version \
+            -v "$(cargo r -- mutants --version | cut -d \  -f 2)" \
+            -r ${{ inputs.release }} \
+            -p cargo-mutants \
+            -e Cargo.lock \
+            -e Cargo.toml \
+            -e CITATION.cff
```

Comment:
This is the actual call to Aeruginous.  Here, we specify the files which shall receive the version increment (and, in case of the CITATION.cff (#212), also the release date update), the current project version (extracted using the compiled binary), and finally the SemVer release level which is forwarded from the workflow invocation.

At this point, further calls to Aeruginous can be added since it is not capable of incrementing version numbers.  Here are some further available features:

- automatic CHANGELOG updates from the Git history;
- mirroring the project description from the crate's root into README to ensure a Single Point of Truth (SPOT) with all Rust code being treated as documentation tests to be executed;
- erasing Carriage Return Line Feeds (CRLFs) to ensure a uniform code style;
- linting the code with lints beyond Clippy's scope; and
- updating CFF-cited dependencies in the CITATION.cff's `references` section.